srcpkgs/libmt32emu/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'libmt32emu'
+pkgname=libmt32emu
+version=2.5.1
+revision=1
+_version_string=${version//./_}
+wrksrc=munt-libmt32emu_${_version_string}
+build_wrksrc=mt32emu
+build_style=cmake
+makedepends="$(vopt_if libsoxr libsoxr-devel)
+ $(vopt_if libsamplerate libsamplerate-devel)"
+short_desc="Library to emulate Roland MT-32, CM-32L and LAPC-I synthesizer
+ modules"
```

Comment:
```suggestion
short_desc="Emulate Roland MT-32, CM-32L and LAPC-I synthesizer modules"
```

If `short_desc` is big enough to need to be split, it should be cut down instead :)

Reply:
Done